src/App.js

```diff
@@ -80,8 +108,39 @@ class App extends Component {
     });
   };
 
+  openSnackBar = ({
+    snackBarMessage,
+    snackBarDuration = 4000,
+    snackBarActionHandler,
+    snackBarAction,
+  }) => {
+    this.setState({
+      snackBarOpen: true,
+      snackBarMessage,
+      snackBarDuration,
+      snackBarActionHandler,
+      snackBarAction,
+    });
+  };
+
+  closeSnackBar = () => {
+    this.setState({
+      snackBarOpen: false,
+      message: '',
```

Comment:
Also reset the other state variables of snackbar. Otherwise it will cause bugs.

Eg - Suppose you show a snackbar with an action and it auto-closes. Now, even without attaching a snackbar handler to next snackbar messages, the action handler would still be present.